charts/scalardb-graphql/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "scalardb-graphql.fullname" . }}-test-connection"
+  labels:
+    {{- include "scalardb-graphql.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args: ['{{ include "scalardb-graphql.fullname" . }}:{{ .Values.service.port }}']
```

Comment:
Sorry, but I just left the `test-connection.yaml` file that the `helm create` command generated as is. https://helm.sh/docs/topics/chart_tests/

Adding the path to the URL is ok to fix the test to check if the path is working. But if we have tests, maybe more meaningful tests should be run. Since other charts don't have any tests yet, I'm thinking of removing the test file.

Reply:
> Sorry, but I just left the test-connection.yaml file that the helm create command generated as is. https://helm.sh/docs/topics/chart_tests/
> 
> Adding the path to the URL is ok to fix the test to check if the path is working. 

I see. I understood.
Thank you for confirmation!

> But if we have tests, maybe more meaningful tests should be run. Since other charts don't have any tests yet, I'm thinking of removing the test file.

OK. I agree with removing the test file.

Reply:
Thank you, removed in 66e4615.